<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>638</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>474</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{border-image: url(:/main-bg.jpg);}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>449</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_userWelcome">
       <property name="styleSheet">
        <string notr="true">font-family: Arial, sans-serif;
font-size: 18px;
font-weight: bold;
color: #333;
background-color: #fff;
text-align: center;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_addFriend">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #eee;
color: black;
border: 1px solid #555;
padding: 10px 20px;
font-size: 16px;
font-weight: bold;</string>
         </property>
         <property name="text">
          <string>添加好友</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_deleteFriend">
         <property name="styleSheet">
          <string notr="true">background-color: #eee;
color: black;
border: 1px solid #555;
padding: 10px 20px;
font-size: 16px;
font-weight: bold;</string>
         </property>
         <property name="text">
          <string>删除好友</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">background-color: #eee;
color: black;
border: 1px solid #555;
padding: 10px 20px;
font-size: 16px;
font-weight: bold;</string>
         </property>
         <property name="text">
          <string>接收文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_sendFile">
         <property name="styleSheet">
          <string notr="true">background-color: #eee;
color: black;
border: 1px solid #555;
padding: 10px 20px;
font-size: 16px;
font-weight: bold;</string>
         </property>
         <property name="text">
          <string>发送文件</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font-family: Arial, sans-serif;
font-size: 18px;
font-weight: bold;
color: #333;
background-color: #fff;
text-align: center;</string>
       </property>
       <property name="text">
        <string>Tips:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font-family: Arial, sans-serif;
font-size: 18px;
font-weight: bold;
color: #333;
background-color: #fff;
text-align: center;</string>
       </property>
       <property name="text">
        <string>联系人列表:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget_friendList"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
